### PR TITLE
Losing sanity.

### PR DESCRIPTION
**Pull request by patrykferenc**

### CODE REVIEW

**Comment by legeof008** [comment]
image

File: Slownik.xml [hunk 1]
```diff
@@ -1,28 +1,44 @@
 <?xml version="1.0" encoding="windows-1252"?>
-<XMI xmi.version="1.1" xmlns:UML="omg.org/UML1.3" timestamp="2023-05-14 23:30:35">
+<XMI xmi.version="1.1" xmlns:UML="omg.org/UML1.3" timestamp="2023-05-21 22:27:07">
+<!-- 
+░░░░░▄▄▄▄▀▀▀▀▀▀▀▀▄▄▄▄▄▄░░░░░░░
```
Comment (legeof008) [inline Slownik.xml:4]: Looks fine to me

**Review by legeof008 — APPROVED** [review]

File: Slownik.xml [hunk 2]
```diff
@@ -1,28 +1,44 @@
...
+█░▒█▒▄░▀▄▄▄▀░░░░░░░░█░░░▒▒▒▒▒░█
+█░▒█░█▀▄▄░░░░░█▀░░░░▀▄░░▄▀▀▀▄▒█
+░█░▀▄░█▄░█▀▄▄░▀░▀▀░▄▄▀░░░░█░░█░
+░░█░░░▀▄▀█▄▄░█▀▀▀▄▄▄▄▀▀█▀██░█░░
+░░░█░░░░██░░▀█▄▄▄█▄▄█▄████░█░░░
+░░░░█░░░░▀▀▄░█░░░█░█▀██████░█░░
+░░░░░▀▄░░░░░▀▀▄▄▄█▄█▄█▄█▄▀░░█░░
+░░░░░░░▀▄▄░▒▒▒▒░░░░░░░░░░▒░░░█░
+░░░░░░░░░░▀▀▄▄░▒▒▒▒▒▒▒▒▒▒░░░░█░
+░░░░░░░░░░░░░░▀▄▄▄▄▄░░░░░░░░█░░
```
Comment (legeof008) [inline Slownik.xml:18]: What an agile change

**Review by legeof008 — APPROVED** [review]

File: Slownik.xml [hunk 3]
```diff
@@ -1,28 +1,44 @@
 <?xml version="1.0" encoding="windows-1252"?>
-<XMI xmi.version="1.1" xmlns:UML="omg.org/UML1.3" timestamp="2023-05-14 23:30:35">
+<XMI xmi.version="1.1" xmlns:UML="omg.org/UML1.3" timestamp="2023-05-21 22:27:07">
+<!-- 
+░░░░░▄▄▄▄▀▀▀▀▀▀▀▀▄▄▄▄▄▄░░░░░░░
+░░░░░█░░░░▒▒▒▒▒▒▒▒▒▒▒▒░░▀▀▄░░░░
+░░░░█░░░▒▒▒▒▒▒░░░░░░░░▒▒▒░░█░░░
+░░░█░░░░░░▄██▀▄▄░░░░░▄▄▄░░░░█░░
```
Comment (legeof008) [inline Slownik.xml:7]: I am loosing my goddamn mind

File: Slownik.xml [hunk 4]
```diff
@@ -1,28 +1,44 @@
...
+░▄▀▒▄▄▄▒░█▀▀▀▀▄▄█░░░██▄▄█░░░░█░
+█░▒█▒▄░▀▄▄▄▀░░░░░░░░█░░░▒▒▒▒▒░█
+█░▒█░█▀▄▄░░░░░█▀░░░░▀▄░░▄▀▀▀▄▒█
+░█░▀▄░█▄░█▀▄▄░▀░▀▀░▄▄▀░░░░█░░█░
+░░█░░░▀▄▀█▄▄░█▀▀▀▄▄▄▄▀▀█▀██░█░░
+░░░█░░░░██░░▀█▄▄▄█▄▄█▄████░█░░░
```
Comment (legeof008) [inline Slownik.xml:13]: image

**Review by legeof008 — APPROVED** [review]

**Review by mciekn — APPROVED** [review]

**Pull request merged**